cli/sub_command.go

```diff
@@ -309,6 +314,30 @@ func (c *subCmd) validateInputs(nc *nats.Conn) error {
 		}
 	}
 
+	isWorkQueue, err := iu.IsWorkQueue(c.stream, nc)
+	if err != nil {
+		return err
+	}
+	c.direct = c.direct || isWorkQueue
+
+	if c.direct {
+		if c.stream == "" {
+			return fmt.Errorf("cannot enable direct gets: stream name is required")
```

Comment:
hmm, could make this work, there's a API for finding a stream hosting a subject. `jsm. StreamNames`